lib/_007/Runtime.pm6

```diff
@@ -15,16 +15,20 @@ class _007::Runtime {
     has $.builtin-frame;
     has $!say-builtin;
     has $!prompt-builtin;
+    has $!exit-builtin;
+    has $.exit-code;
 
     submethod BUILD(:$!input, :$!output) {
+        $!builtin-opscope = opscope();
         $!builtin-frame = Val::Object.new(:properties(
             :outer-frame(NO_OUTER),
             :pad(builtins-pad()))
         );
         @!frames.push($!builtin-frame);
         $!say-builtin = builtins-pad().properties<say>;
         $!prompt-builtin = builtins-pad().properties<prompt>;
-        $!builtin-opscope = opscope();
+        $!exit-builtin = builtins-pad().properties<exit>;
+        $!exit-code = 0;
```

Comment:
Is this the default (the zero I mean)? In that case, how about adding a constant with a self-explaining name with the value 0 and apply that var to exit-code? Perhaps everyone know that 0 is default here?

Reply:
Agreed about a constant being an improvement. Fixed in cf4273a0ec8f3970fcb70277ef12e2c0a11d309f.